source/PlayState.hx

```diff
@@ -1,20 +1,35 @@
 package;
 
 import flixel.FlxState;
-import haxe.display.Protocol.HaxeResponseErrorData;
 import heroes.Hero;
+import heroes.ScreensaverHero;
+import heroes.ControllableHero;
 
 class PlayState extends FlxState
 {
 	override public function create()
 	{
 		super.create();
+		var text = new flixel.text.FlxText(0, 0, 0, "How are ya, hero?", 32);
+		text.screenCenter();
+		add(text);
 
 		var myHero = new Hero(200, 400);
+		add(myHero);
+		var ScreensaverHero1 = new ScreensaverHero(100, 200);
+		add(ScreensaverHero1);
+		var ScreensaverHero2 = new ScreensaverHero(200, 100);
+		add(ScreensaverHero2);
+		var ScreensaverHero3 = new ScreensaverHero(50, 20);
+		add(ScreensaverHero3);
+		var ScreensaverHero4 = new ScreensaverHero(80, 80);
+		add(ScreensaverHero4);
```

Comment:
Loops are your friend here, something like:
```
for (i in 0...4) {
    add(new ScreensaverHero(i * 50, i * 50))
}
```

This gets the job done, which is a fine result when you're under time pressure, but it's not hard to see the pain that repetition brings. That, and it doesn't hurt to get more comfortable with the fundamentals of the Haxe language.

Reply:
No action required here, but optionally swap to a loop if you want the practice.